index.html

```diff
@@ -0,0 +1,304 @@
+            <li>Ruby on rails</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+            <li>RSPec</li>
+          </ul>
+          <div class="skillotherheading">
+            <img src="/images/circle.png" alt="" />
+            <h1>Skills</h1>
+          </div>
+          <ul class="skilOther">
+            <li>Codekit</li>
+            <li>github</li>
+            <li>Codepen</li>
+            <li>gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </section>
+      <section class="details">
+
+        <div class="contact-p">
+             <p>
+              I'm always interested in hearing about new projects, so if you'd like
+              to chat please get in touch.
+            </p>
+        </div>
+        <form action="https://formspree.io/f/xyyvjvdn" method="POST">
+          <div class="ContactForm">
+              <div class="inputText">
+                      <input type="text" name="name" placeholder="Full name"
+                       required="" maxlength="30" type="text" class="input_t">
+                       <div class="line-"></div>
+                </div>
+                
+    
+              <div class="inputText">
+                  <input
+                   type="email"name="email" required=""
+                    placeholder="Email address" class="input_t">
+                      
+               </div>
+               
+    
+              <div class="inputTextarea">
+                  <textarea
+                   id="comment"
+                   name="comment"
+                   cols="50" rows="10" maxlength="500" required=""
+                   placeholder="Enter text here"
+                   type="text" class="text-area"></textarea>
+                </div>
+                
+                <div class="contact-button">
+                      <input type="Submit" value="Get In Touch" class="btn-touch">
+                </div>
+            </div>
+        </form>
+        <div class="cross"></div>
+        <div class="contact-social">
+            <div class="contact-social-icon">
+              
+                <a href="https://github.com/Robison-faike123">
+                  <img class="image"src="images/help.png" alt="-GitHub" />
+                </a>
+            
+                <a href="https://www.linkedin.com/">
+                  <img class="image"src="./images/Disabled.png" alt="Linkedin" />
+                </a>
+            
+                <a href="#">
+                  <img class="image"src="./images/strugle.png" alt="Angelist" />
+                </a>
+              
+                <a href="https://twitter.com">
+                  <img class="image"src="./images/twitter.png" alt="Twitter" />
+                </a>
+              
+                <a href="#">
+                  <img class="image" src="./images/medium.png" alt="Medium" />
+                </a>
+            </div> <br>
+          <div class="line"></div>
+        </div>
```

Comment:
- [ ] to be semantically correct kindly move the `contact-social` div outside of `details` section, and wrap it inside a `footer`tag